coolfacts/entrypoint/main.go

```diff
@@ -4,54 +4,23 @@ import (
 	"context"
 	"fmt"
 	"github.com/FTBpro/go-workshop/coolfacts/entrypoint/fact"
-	"github.com/FTBpro/go-workshop/coolfacts/entrypoint/mentalfloss"
+	"github.com/FTBpro/go-workshop/coolfacts/entrypoint/inmem"
 	"github.com/FTBpro/go-workshop/coolfacts/entrypoint/myhttp"
+	"github.com/FTBpro/go-workshop/coolfacts/entrypoint/providers/mentalfloss"
 	"log"
 	"net/http"
-	"time"
```

Comment:
better practice use goimports -w  
This will order the imports alphabatically and separate the go imports from the others